test/error_test.py

```diff
@@ -61,7 +61,7 @@
 >>> g.plot(color=2, bgcolor=2)
 Traceback (most recent call last):
         ...
-samila.errors.samilaPlotError: Given color/bgcolor type is not supported.
+samila.errors.samilaPlotError: Plotting process can't be Done because data1 is empty. Use generate method first.
```

Comment:
`Done` --> `done`
Alos how are these changed without any changes in `params.py`? Either now or then tests should be failed, right?

Reply:
Due to the `IGNORE_EXCEPTION_DETAIL` flag in the `pytest.ini` file, all exception messages were being ignored.

Reply:
> `Done` --> `done` 

fix in https://github.com/sepandhaghighi/samila/pull/238/commits/c23ef8dbb76bb8f719c59364403173261276c64e